jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -2375,6 +2375,16 @@ public boolean isInterface() {
  */
 public native boolean isPrimitive();
 
+/**
+ * Answers true if the receiver represents a primite class type.
```

Comment:
The indentation seems off here

Reply:
It's the same indentation as the above code.

Reply:
uhm, that must have been the old style. Can you make it look more like `getAnnotation` below. Dont need to worry about `@since ...`